src/main/java/frc/robot/subsystems/intake/pivot/PivotState.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems.intake.pivot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public enum PivotState {
```

Comment:
I've asked to program the [Elevator states as a part of the superstructure](https://github.com/GreenBlitz/2024-Offseason-robot/pull/3#discussion_r1783258236), I think it impiles to this PR too.

Reply:
its ok, u can aswell (im guessing its shahar) start the states, just make sure to finish the elvator first